File: shanny-blog-java/blog-pojo/src/main/java/com/shanny/vo/UserInfoVO.java
package com.shanny.vo;

import com.shanny.entity.UserDetails;
import com.shanny.enums.UserEnum;
import io.swagger.v3.oas.annotations.media.Schema;
import lombok.AllArgsConstructor;
import lombok.Builder;
import lombok.Data;
import lombok.NoArgsConstructor;

import java.time.LocalDateTime;

@Data
@Builder
@NoArgsConstructor
@AllArgsConstructor
@Schema(description = "用户基础信息")
public class UserInfoVO {
    private String uuid;
    private String userId;
    private String mobile;
    private UserEnum.UserStatus status;
    private UserEnum.UserType type;
    private LocalDateTime createTime;
    private LocalDateTime updateTime;
    private LocalDateTime lastLoginTime;
    private UserDetails userDetails;
}

File: shanny-blog-net/blog-pojo/Vos/UserInfoVO.cs
using blog_common.Enums;
using blog_pojo.Entities;

namespace blog_pojo.Vos
{
    public class UserInfoVO
    {
        public string Uuid { get; set; } = string.Empty;
        public string UserId { get; set; } = string.Empty;
        public string Mobile { get; set; } = string.Empty;
        public UserStatus Status { get; set; }
        public UserType Type { get; set; }
        public DateTime? CreateTime { get; set; }
        public DateTime? UpdateTime { get; set; }
        public DateTime? LastLoginTime { get; set; }
        public UserDetails? UserDetails { get; set; }
    }
}
